examples/protobufjs/package.json

```diff
@@ -0,0 +1,16 @@
+{
+	"name": "jazzerjs-promise-example",
```

Comment:
Wrong package name. Also leads to test warnings, due to the duplication.

Reply:
Whoops. Done.